docs/protocol/src/community_pool/action/deposit.md

```diff
@@ -1 +0,0 @@
-# CommunityPoolDeposit
```

Comment:
The audit (correctly) pointed out that we lack complete documentation of the protocol, and have these stub pages in the protocol spec. We _should_ remove pages that are no longer relevant, but I think in every case the stub pages are relevant, i.e. the `CommunityPoolDeposit` does exist as an action, and as such should be ideally documented in the protocol specification. So I think the action here is to fill these pages out (not in this PR because that's a significant chunk of work) instead of deleting them. One thing we could do is just make a list of pages that should be filled out in an issue.

Reply:
sounds great, will revert the deleted pages.

Reply:
references https://github.com/penumbra-zone/penumbra/issues/4516